<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game3d</class>
 <widget class="QWidget" name="Game3d">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonPause">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>41</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(170, 85, 0); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(0, 170, 255); }</string>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBarTimer">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>41</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>60</number>
   </property>
   <property name="value">
    <number>60</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="format">
    <string>%p sec</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBackToMenu">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>40</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(170, 85, 0); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(0, 170, 255); }</string>
   </property>
   <property name="text">
    <string>BackToMenu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCheat">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(170, 85, 0); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(0, 170, 255); }</string>
   </property>
   <property name="text">
    <string>Cheat</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
